src/frontend/js/utils/test/wrappers/ReactQueryAppWrapper.tsx

```diff
@@ -0,0 +1,16 @@
+import { PropsWithChildren } from 'react';
+import { IntlWrapper } from './IntlWrapper';
+import { ReactQueryWrapper } from './ReactQueryWrapper';
+import { AppWrapperProps } from './types';
+
+export const ReactQueryAppWrapper = ({
+  children,
+  intlOptions,
+  queryOptions,
+}: PropsWithChildren<AppWrapperProps>) => {
+  return (
+    <IntlWrapper {...(intlOptions || { locale: 'en' })}>
+      <ReactQueryWrapper {...(queryOptions || {})}>{children}</ReactQueryWrapper>
+    </IntlWrapper>
+  );
+};
```

Comment:
Just a suggestion to improve readability.

```suggestion
export const ReactQueryAppWrapper = ({
  children,
  intlOptions = { locale: 'en' },
  queryOptions = {},
}: PropsWithChildren<AppWrapperProps>) => {
  return (
    <IntlWrapper {...intlOptions}>
      <ReactQueryWrapper {...queryOptions}>{children}</ReactQueryWrapper>
    </IntlWrapper>
  );
};
```